action.yml

```diff
@@ -71,7 +71,11 @@ runs:
         # NOTE: Sourced, not executed as a script.
         source "${{ github.action_path }}/setup/venv.bash"
 
-        ${{ github.action_path }}/action.py "${{ inputs.inputs }}"
+        if [[ "${{runner.os}}" == "Windows" ]]; then
+          python "${{ github.action_path }}/action.py" "${{ inputs.inputs }}"
+        else
+          ${{ github.action_path }}/action.py "${{ inputs.inputs }}"
+        fi
```

Comment:
oh that would be fine then. I just thought about not changing the existing lines to not break anything since i don't have a Linux environment to test.

Reply:
I updated the code and removed it